backend/iot_mobile/src/main/kotlin/edu/pwr/iotmobile/websocket/IncomingMessageWebSocketHandler.kt

```diff
@@ -0,0 +1,65 @@
+package edu.pwr.iotmobile.websocket
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule
+import edu.pwr.iotmobile.rabbit.RabbitListener
+import edu.pwr.iotmobile.service.IncomingMessageService
+import lombok.extern.slf4j.Slf4j
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import org.springframework.stereotype.Component
+import org.springframework.web.socket.CloseStatus
+import org.springframework.web.socket.TextMessage
+import org.springframework.web.socket.WebSocketSession
+import org.springframework.web.socket.handler.TextWebSocketHandler
+import reactor.core.Disposable
+
+@Component
+@Slf4j
+class IncomingMessageWebSocketHandler(
+    val incomingMessageService: IncomingMessageService,
+    val rabbitListener: RabbitListener
+): TextWebSocketHandler() {
+    private val objectMapper = ObjectMapper().registerModule(JavaTimeModule())
+    private val logger: Logger = LoggerFactory.getLogger("Websocket message handler")
+    private var queues : MutableSet<String> = LinkedHashSet()
+
+
+
+    @Throws(Exception::class)
+    override fun handleTextMessage(session: WebSocketSession, message: TextMessage) {
+        val queueList: List<String> = registerQueues(message)
```

Comment:
The message could be a stringified JSON with the list of topic unique names, instead of a string separated by commas